contrib/kind.sh

```diff
@@ -37,6 +39,10 @@ usage()
     echo "                             DEFAULT: Don't install ingress components."
     echo "-n4 | --no-ipv4              Disable IPv4. DEFAULT: IPv4 Enabled."
     echo "-i6 | --ipv6                 Enable IPv6. DEFAULT: IPv6 Disabled."
+    echo "-mt | --num-masters          Number of master nodes in HA mode. Ignored in"
```

Comment:
let's not support variable number of masters, either 3 for HA or 1 for no-ha

Reply:
OK, I'll remove.

Reply:
Done